### PR TITLE
 [15.0][MIG] sale_product_multi_add

### PR DESCRIPTION
**Pull request by juanpgarza**
Standard migration

### CODE REVIEW

**Comment by rafaelbn** [comment]
@rousseldenis this is very well know by you 😄 a quick review? Thank you!

File: sale_product_multi_add/__manifest__.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "author": "ACSONE SA/NV, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/sale-workflow",
     "category": "Sale Management",
-    "version": "14.0.1.0.1",
+    "version": "15.0.1.0.1",
```
Comment (rousseldenis) [inline sale_product_multi_add/__manifest__.py]: @juanpgarza The good flow for migration is to reset major/minor/patch to : 1.0.0.

**Review by rousseldenis — APPROVED** [review]
Minor comment

**Review by rafaelbn — APPROVED** [review]

**Pull request merged**